src/handlebars/index.handlebars

```diff
@@ -13,10 +13,30 @@
     <h2 id="dl-text" class="animated fadeIn">Downloads</h2>
 
     <div id="error-container"></div>
-    <div id='loading'><img src='dist/assets/loading_dots.gif' height='40' width='40'></div>
 
-    <select id="variant-selector"></select>
+    <div class="btn-container">
+      <form id="jdk-selector" class="btn-form">
+        <h3>1. Choose a Version</h3>
+      </form>
+      <form id="jvm-selector" class="btn-form">
+        <h3>2. Choose a JVM
+          <span class="help-jvm">Help Me Choose</span>
+          <div class="popup-content" id="jvm-popup">
+            <div class="popup-heading">
+              <h4>Choosing the right JVM</h4>
+            </div>
+            <div class="popup-text">
```

Comment:
See below for some suggested PR updates to make the help text more 'symmetrical', and link to the Hotspot VM pages for help.

Reply:
FWIW - I think they're all good suggestions